openpype/hosts/maya/plugins/load/load_arnold_standin.py

```diff
@@ -6,22 +6,34 @@
 from openpype.settings import get_project_settings
 from openpype.pipeline import (
     load,
+    legacy_io,
     get_representation_path
 )
 from openpype.hosts.maya.api.lib import (
-    unique_namespace, get_attribute_input, maintained_selection
+    unique_namespace,
+    get_attribute_input,
+    maintained_selection,
+    convert_to_maya_fps
 )
 from openpype.hosts.maya.api.pipeline import containerise
 
 
 def is_sequence(files):
     sequence = False
-    collections, remainder = clique.assemble(files)
+    if len(files) == 1:
+        collections, remainder = clique.assemble(files, minimum_items=1)
+    else:
+        collections, remainder = clique.assemble(files)
     if collections:
         sequence = True
-
     return sequence
 
+def get_fps(standin_shape):
```

Comment:
You dont need this function if you get the fps data from the published object through the version data.